snapcraft/meta.py

```diff
@@ -100,13 +100,39 @@ def _setup_assets(meta_dir, config_data):
         _setup_config_hook(hooks_dir, config_data['config'])
 
     if 'license' in config_data:
+        logger.warning("DEPRECATED: 'license' defined in snapcraft.yaml")
         license_path = os.path.join(meta_dir, 'license.txt')
-        shutil.copyfile(config_data['license'], license_path)
+        os.link(config_data['license'], license_path)
 
     if 'icon' in config_data:
+        logger.warning("DEPRECATED: 'icon' defined in snapcraft.yaml")
```

Comment:
Ah, I didn't realize this was actually going away. So this will become store-only?